matlab-hello-world-model/README.md

```diff
@@ -0,0 +1,106 @@
+# Matlab Hello World Model
+
+This is a very simple Model written in Matlab to demonstrate the
+process of Dockerising a Matlab Model for upload onto DAFNI.
+
+The Model itself reads a csv file from a Dataset that has been
+uploaded to DAFNI, an example of what the csv file looks like,
+`dataset.csv`, is included in this folder. The Model then reads
+a parameter from the environment variables, parameters values
+are passed to Models via environment variables on DAFNI. The 
+parameter should contain an integer, the Model will add this
+integer to the Matrix that is stored in the csv file. Finally
+the Model will write the new Matrix to a file as an output.
+
+File Structure:
+
+- _[Dockerfile](./Dockerfile)_ - Builds the container that will be
+   run by DAFNI
+- _[model_definition.yaml](./model_definition.yaml)_ - A 
+   machine-readable file used to define the Model.
+   This information will be shown to other users who may wish to 
+   use your Model.
+- _[dataset.csv](./dataset.csv)_ - an example csv file to run
+   the Model with, on DAFNI this file will be provided by a 
+   dataset.
+- _[helloWorld.m](./helloWorld.m)_ - the Matlab Model code
+- _[run_model.sh](./src/run_model.sh)_ - a script that copies
+   the input dataset files from DAFNI's data directory to the 
+   folder the Model expects the files to be in, then runs the 
+   Model and then copies the output file to DAFNI's data directory
+   so DAFNI can find it.
+
+As this Model uses Matlab the process required to build a Docker
+image that can run the Model is more complicated than for the 
+other example Models. In fact the process we use is to generate
+a Docker image from Matlab itself and then use that as a base
+for the Dockerfile in this folder.
+
+To start with you will need to have a Linux machine with both
+Matlab and Docker installed. The reason you need a Linux machine
+is because the Matlab commands needed to produce the Docker image
+only work on Linux, as you can see from their
+[documentation](https://uk.mathworks.com/help/compiler/package-matlab-standalone-applications-into-docker-images.html).
+While you could use this documentation to work out how to dockerise
+this Model for yourself I would recommend continuing on with this
+README as we have already tweaked the commands to work for this
+case.
```

Comment:
```suggestion
To start with, you will need to have a Linux machine with both
Matlab and Docker installed. The reason you need a Linux machine
is because the Matlab commands needed to produce the Docker image
only work on Linux, as you can see from their
[documentation](https://uk.mathworks.com/help/compiler/package-matlab-standalone-applications-into-docker-images.html).
While you could use this documentation to work out how to dockerise
this Model for yourself I would recommend continuing on with this
README as we have already tweaked the commands to work for DAFNI's
specific use-case.
```